src/browser/documentation/sidebar-guides/cypher.tsx

```diff
@@ -19,189 +19,175 @@
+          <li>
+            <code>WHERE</code> filters the query.
+          </li>
+          <li>
+            <code>ee.name = 'Emil'</code> compares name property to the value{' '}
+            <code>Emil</code>.
+          </li>
+          <li>
+            <code>RETURN</code> returns particular results.
+          </li>
+        </ul>
+      </li>
+      <li>
+        Run the Cypher code by clicking the <b>Run</b> button.
+      </li>
+      <br />
+    </ol>
   </BuiltInGuideSidebarSlide>,
-  <BuiltInGuideSidebarSlide key="forth">
-    <h3>CREATE more</h3>
-    <p className="lead">Nodes and relationships</p>
+  <BuiltInGuideSidebarSlide key="s4">
+    <h3>CREATE more data</h3>
+    <p className="lead">
+      <em>Nodes and relationships</em>
+    </p>
     <p>
-      <code>CREATE</code>
-      clauses can create many nodes and relationships at once.
+      The <code>CREATE</code> clause can create many nodes and relationships at
+      once.
     </p>
     <pre className="pre-scrollable code runnable">
-      {`MATCH (ee:Person) WHERE ee.name = "Emil"
-CREATE (js:Person { name: "Johan", from: "Sweden", learn: "surfing" }),
-(ir:Person { name: "Ian", from: "England", title: "author" }),
-(rvb:Person { name: "Rik", from: "Belgium", pet: "Orval" }),
-(ally:Person { name: "Allison", from: "California", hobby: "surfing" }),
+      {`MATCH (ee:Person) WHERE ee.name = 'Emil'
+CREATE (js:Person { name: 'Johan', from: 'Sweden', learn: 'surfing' }),
+(ir:Person { name: 'Ian', from: 'England', title: 'author' }),
+(rvb:Person { name: 'Rik', from: 'Belgium', pet: 'Orval' }),
+(ally:Person { name: 'Allison', from: 'California', hobby: 'surfing' }),
 (ee)-[:KNOWS {since: 2001}]->(js),(ee)-[:KNOWS {rating: 5}]->(ir),
 (js)-[:KNOWS]->(ir),(js)-[:KNOWS]->(rvb),
 (ir)-[:KNOWS]->(js),(ir)-[:KNOWS]->(ally),
 (rvb)-[:KNOWS]->(ally)`}
     </pre>
+    <br />
   </BuiltInGuideSidebarSlide>,
-  <BuiltInGuideSidebarSlide key="fifth">
-    <h3>Pattern matching</h3>
-    <p className="lead">Describe what to find in the graph</p>
+
+  <BuiltInGuideSidebarSlide key="s5">
+    <h3>MATCH patterns</h3>
+    <p className="lead">
+      <em>Describe what to find in the graph</em>
+    </p>
     <p className="summary">
       For instance, a pattern can be used to find Emil's friends:
     </p>
     <pre className="pre-scrollable code runnable">
       {`MATCH (ee:Person)-[:KNOWS]-(friends)
-WHERE ee.name = "Emil" RETURN ee, friends`}
+WHERE ee.name = 'Emil' RETURN ee, friends`}
     </pre>
-    <ul>
+    <ul style={{ marginLeft: '10px', paddingLeft: '10px' }}>
       <li>
-        <code>MATCH</code>
-        clause to describe the pattern from known Nodes to found Nodes
+        <code>MATCH</code> describes what nodes will be retrieved based upon the
+        pattern.
       </li>
       <li>
-        <code>(ee)</code>
-        starts the pattern with a Person (qualified by WHERE)
+        <code>(ee)</code> is the anchor node that will be found based upon the{' '}
```

Comment:
"anchor node", is this a new term? I would call it a reference to the node.

https://neo4j.com/docs/cypher-manual/4.3/syntax/variables/